src/Approximations/overapproximate.jl

```diff
@@ -1963,3 +1963,59 @@ function overapproximate(X::Intersection{N, <:Hyperplane, <:AbstractZonotope},
                          dirs::Type{<:AbstractDirections}) where {N<:Real}
     return overapproximate(X.Y ∩ X.X, dirs(dim(X)))
 end
+
+# ===========================================================
+# Functionality that requires IntervalConstraintProgramming
+# ===========================================================
+
+# function to be loaded by Requires
+function load_paving_overapproximation()
+
+return quote
+
+using .IntervalConstraintProgramming: Paving
+
+
+"""
+    overapproximate(p::Paving{L, N}, dirs::AbstractDirections{N, VN}) where {L, N, VN}
+
+Overapproximation of a Paving-type set representation using an HPolytope.
+
+### Input
+
+- `p`    -- type of pave representation
+- `dirs` -- directions of the HPolytope
+
+### Output
+
+An overapproximation of a Paving-type set using an HPolytope with constraints in direction dirs.
+
+### Algorithm
+
+This function takes the union of the elements in the boundary of p, first
+converted into hyperrectangles, and then calculates the support function of the
+set along each  direction in dirs, to compute the HPolytope constraints.
+
+### Requires IntervalConstraintProgramming
+"""
+function overapproximate(p::Paving{L, N}, dirs::AbstractDirections{N, VN}) where {L, N, VN}
```

Comment:
this function needs to be added to the docs..?

Reply:
Not in this case. We just let `Documenter` build [all docs](https://github.com/JuliaReach/LazySets.jl/blob/master/docs/src/lib/approximations.md#overapproximations) for the function `overapproximate`.